app/frontend/src/components/admin/inspection/LocationEdit.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <v-dialog v-model="dialog" max-width="800px">
+    <template v-slot:activator="{ on }">
+      <!-- <v-btn color="primary" dark v-on="on">Open Dialog</v-btn> -->
+      <v-btn v-if="true" v-on="on" color="primary" class="my-3" small>Edit Submission</v-btn>
+    </template>
+    <v-card>
+      <v-form ref="form" v-model="updateValid">
+        <v-card-title>
+          <span class="headline">
+            Editing submission from:
+            <strong>{{businessName}}</strong>
+          </span>
+        </v-card-title>
+        <v-card-text>
+          <v-container class="pa-0">
+
+            <!-- child Location form component -->
+            <Location :reviewMode="reviewMode"/>
+
+          </v-container>
+        </v-card-text>
+        <v-card-actions class="pa-3 pb-10">
+          <v-btn color="primary" @click="update">Save</v-btn>
+          <v-btn text @click="cancelUpdate">Cancel</v-btn>
+        </v-card-actions>
+      </v-form>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+import Vuex from 'vuex';
+import { mapGetters } from 'vuex';
+
+import Location from '@/components/form/Location.vue';
+import { SilvipcRoles } from '@/utils/constants';
+
+/* eslint-disable */
+import ipcService from '../../../services/ipcService';
+
+const localStore = new Vuex.Store({
+  state: {
+    // copy location state from global store
+    location: this.location
+  },
+  actions: {
+    async updateForm({ commit, state }) {
+
+      console.log(localStore.location);
+
+      try {
+        const body = {
+          location: localStore.location
+        };
+        const response = await ipcService.updateIPCContent(body);
+
+        if (!response.data) {
+          throw new Error('No response data from API while submitting update');
+        }
+      } catch (error) {
+        console.error(`Error submitting update: ${error}`); // eslint-disable-line no-console
+      }
+    }
+
+  }
+})
+
+export default {
+  name: 'LocationEdit',
+  components: {
+    Location,
+  },
+  props: {
+    ipcPlanId: {
+      required: true,
+      type: String
+    },
+  },
+  data() {
+    return {
+      showEditBtn: false,
+      dialog: false,
+      updateValid: false,
+      reviewMode: false,
+      startDateMenu: false,
+      endDateMenu: false,
+    };
+  },
+  computed: {
+    ...mapGetters('form', ['business', 'ipcPlan', 'location', 'submissionComplete']),
+    ...mapGetters('auth', ['hasSilvipcRoles']),
+
+    // Business data
+    businessName: {
+      get() { return this.business.name; },
+    },
```

Comment:
i'm still populating the update form with data from the form store,
but i've taken out all the setters.

Reply:
Following the previous comment, you want the following Vue options in this module:
```js
  beforeDestroy() {
    this.$store.unregisterModule('edit');
  },
  created() {
    if(this.$store.hasModule('edit')) {
      this.$store.unregisterModule('edit');
    }
    this.$store.registerModule('edit', form);
  }
```
This would be how you lazy-load the form module into vuex, while putting it in a completely different namespace than 'form' to avoid editing the 'form' namespace state.